config.example.json

```diff
@@ -4,7 +4,12 @@
     "memer_secret": "",
     "patreonCampaignID": "",
     "discord_baseURL": "https://discordapp.com/api",
+    "paypalVerifyURL": "https://api.sandbox.paypal.com/v1/notifications/verify-webhook-signature",
```

Comment:
Will this need changed to be not sandbox?

Reply:
@aetheryx

Reply:
in prod. yes